components/Workspaces/WorkspaceHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import { SquareFillIcon } from "@primer/octicons-react";
+import { FaEdit } from "react-icons/fa";
+import Button from "components/atoms/Button/button";
+
+interface WorkspaceHeaderProps {
+  workspace: Workspace;
+}
+
+export const WorkspaceHeader = ({ workspace }: WorkspaceHeaderProps) => {
+  return (
+    <section className="w-full grid grid-cols-[1fr,auto] gap-2">
+      <h1 className="flex gap-2 items-center uppercase text-3xl font-semibold w-max max-w-sxs md:max-w-sm lg:max-w-2xl xl:max-w-4xl">
+        {/* putting a square icon here as a placeholder until we implement workspace logos */}
+        <SquareFillIcon className="w-12 h-12 text-sauced-orange" />
+        <span title={workspace.name} className="whitespace-nowrap overflow-hidden text-ellipsis">
+          {workspace.name}
+        </span>
+      </h1>
+      <Button
```

Comment:
@zeucapua, for the work for accessing settings only if you're the owner, has that been merged? Just asking as beta didn't seem to have a check on the button on whether a workspace could be edited or not.